doc/man1/openssl-req.pod.in

```diff
@@ -250,15 +259,31 @@ This option has been deprecated and has no effect.
 
 =item B<-x509>
 
-This option outputs a self signed certificate instead of a certificate
-request. This is typically used to generate a test certificate or
-a self signed root CA. The extensions added to the certificate
-(if any) are specified in the configuration file. Unless specified
-using the B<-set_serial> option, a large random number will be used for
-the serial number.
+This option outputs a certificate instead of a certificate request.
+This is typically used to generate test certificates.
+
+If an existing request is specified with the B<-in> option, it is converted
+to the a certificate; otherwise a request is created from scratch.
+
+Unless specified using the B<-set_serial> option,
+a large random number will be used for the serial number.
+
+X.509 extensions are not copied from any provided request input file.
```

Comment:
Add `by default`?

Reply:
Good catch, since this behavior has been changed during the history of this PR.
So I've prepended this by "Unless the B<-copy_extensions> option is used,"